### PR TITLE
Use setting to limit context

### PR DESCRIPTION
**Pull request by dylanjcastillo**
Hi @EmilStenstrom,

I created a PR for this [issue](https://github.com/EmilStenstrom/django-components/issues/351), including tests.

Let me know if this is what you had in mind.

Best,
Dylan

### CODE REVIEW

File: django_components/app_settings.py [hunk 1]
```diff
@@ -17,5 +17,9 @@ def LIBRARIES(self):
     def TEMPLATE_CACHE_SIZE(self):
         return self.settings.setdefault("template_cache_size", 128)
 
+    @property
```
Comment (dylanjcastillo) [inline django_components/app_settings.py:20]: @EmilStenstrom I added this to the app_settings, but I'm not sure what's the logic of picking an app vs. a global setting.

I see that in the Middleware you went for a global setting.

**Comment by EmilStenstrom** [comment]
@dylanjcastillo What's the plan if we add another isolation behaviour as per the separate discussion? Here the value is just True/False... Should it be an enum from the getgo?

**Comment by dylanjcastillo** [comment]
Hey @EmilStenstrom, you're right! I updated it to an enum

**Comment by EmilStenstrom** [comment]
Awesome! I really appreciate these PRs @dylanjcastillo! Mind sending an e-mail to emil@emilstenstrom.se. There's some funding available for this project that I want to redirect to you!

**Comment by dylanjcastillo** [comment]
Wow! That's very nice of you. Will reach out. Thank you!

**Pull request merged**